<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogMixture</class>
 <widget class="QDialog" name="DialogMixture">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>558</width>
    <height>302</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Création mixture</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../ressources.qrc">
    <normaloff>:/icones/icon</normaloff>:/icones/icon</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="margin">
    <number>6</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Nom de la mixture :</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineNom"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Création de son :</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboFreq">
       <item>
        <property name="text">
         <string>à chaque note</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>toutes les 3 notes</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>toutes les 6 notes</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QCheckBox" name="checkBouclage">
       <property name="text">
        <string>Bouclage</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QCheckBox" name="checkStereo">
       <property name="text">
        <string>Stéréo</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Divisions</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2" columnstretch="100,0,0,0">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>9</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="verticalSpacing">
         <number>6</number>
        </property>
        <item row="3" column="1" colspan="3">
         <widget class="SpinBoxKey" name="spinDivStart">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximum">
           <number>127</number>
          </property>
          <property name="value">
           <number>36</number>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="pushAddDiv">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>24</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../ressources.qrc">
            <normaloff>:/icones/add</normaloff>:/icones/add</iconset>
          </property>
         </widget>
        </item>
        <item row="0" column="0" rowspan="6">
         <widget class="QListWidget" name="listDivisions">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="pushRemoveDiv">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>24</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../ressources.qrc">
            <normaloff>:/icones/remove</normaloff>:/icones/remove</iconset>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>6</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="3">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="4" column="1" colspan="3">
         <widget class="SpinBoxKey" name="spinDivEnd">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximum">
           <number>127</number>
          </property>
          <property name="value">
           <number>96</number>
          </property>
         </widget>
        </item>
        <item row="2" column="1" colspan="3">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Etendue</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Rangs</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3" columnstretch="100,0,0,0">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>9</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="verticalSpacing">
         <number>6</number>
        </property>
        <item row="1" column="1">
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>6</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="3">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="pushRemoveRank">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../ressources.qrc">
            <normaloff>:/icones/remove</normaloff>:/icones/remove</iconset>
          </property>
         </widget>
        </item>
        <item row="2" column="1" colspan="3">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Type</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1" colspan="3">
         <widget class="QComboBox" name="comboType1">
          <item>
           <property name="text">
            <string>octave</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>quinte</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>tierce</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>septième</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>neuvième</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>onzième</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>treizième</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>quinzième</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>dix-septième</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>dix-neuvième</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>vingt-et-unième</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>vingt-troisième</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>vingt-cinquième</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>vingt-septième</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="4" column="1" colspan="3">
         <widget class="QComboBox" name="comboType2"/>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="pushAddRank">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../ressources.qrc">
            <normaloff>:/icones/add</normaloff>:/icones/add</iconset>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="0" rowspan="6">
         <widget class="QListWidget" name="listRangs">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SpinBoxKey</class>
   <extends>QSpinBox</extends>
   <header>spinboxkey.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lineNom</tabstop>
  <tabstop>checkStereo</tabstop>
  <tabstop>comboFreq</tabstop>
  <tabstop>checkBouclage</tabstop>
  <tabstop>listDivisions</tabstop>
  <tabstop>listRangs</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="../ressources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogMixture</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogMixture</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
